pkg/kubelet/dockershim/docker_service.go

```diff
@@ -254,24 +255,27 @@ func NewDockerService(config *ClientConfig, podSandboxImage string, streamingCon
 	ds.network = network.NewPluginManager(plug)
 	klog.Infof("Docker cri networking managed by %v", plug.Name())
 
-	// NOTE: cgroup driver is only detectable in docker 1.11+
-	cgroupDriver := defaultCgroupDriver
-	dockerInfo, err := ds.client.Info()
-	klog.Infof("Docker Info: %+v", dockerInfo)
-	if err != nil {
-		klog.Errorf("Failed to execute Info() call to the Docker client: %v", err)
-		klog.Warningf("Falling back to use the default driver: %q", cgroupDriver)
-	} else if len(dockerInfo.CgroupDriver) == 0 {
-		klog.Warningf("No cgroup driver is set in Docker")
-		klog.Warningf("Falling back to use the default driver: %q", cgroupDriver)
-	} else {
-		cgroupDriver = dockerInfo.CgroupDriver
-	}
-	if len(kubeCgroupDriver) != 0 && kubeCgroupDriver != cgroupDriver {
-		return nil, fmt.Errorf("misconfiguration: kubelet cgroup driver: %q is different from docker cgroup driver: %q", kubeCgroupDriver, cgroupDriver)
+	if runtime.GOOS == "linux" {
+		// NOTE: cgroup driver is only detectable in docker 1.11+
+		cgroupDriver := defaultCgroupDriver
+		dockerInfo, err := ds.client.Info()
+		klog.Infof("Docker Info: %+v", dockerInfo)
+		if err != nil {
+			klog.Errorf("Failed to execute Info() call to the Docker client: %v", err)
+			klog.Warningf("Falling back to use the default driver: %q", cgroupDriver)
+		} else if len(dockerInfo.CgroupDriver) == 0 {
+			klog.Warningf("No cgroup driver is set in Docker")
+			klog.Warningf("Falling back to use the default driver: %q", cgroupDriver)
+		} else {
+			cgroupDriver = dockerInfo.CgroupDriver
+		}
+		if len(kubeCgroupDriver) != 0 && kubeCgroupDriver != cgroupDriver {
+			return nil, fmt.Errorf("misconfiguration: kubelet cgroup driver: %q is different from docker cgroup driver: %q", kubeCgroupDriver, cgroupDriver)
+		}
+		klog.Infof("Setting cgroupDriver to %s", cgroupDriver)
+		ds.cgroupDriver = cgroupDriver
```

Comment:
Can we log that we are skipping cgroup checks because it's windows ?

That will help make the kubelet logs for windows nodes easier to follow .

Reply:
And also remind us to maybe do some equivalent steps at some point down the road , for windows

Reply:
What would the equivalent steps be for Windows wrt cGroups? Maybe we should open an issue to track that?

Reply:
i don't think there is a need for a note in the code or a separate tracking issue.

there is no "cgroup driver" concept on Windows as there is no "control group management" fragmentation due to multiple solutions.

it has Job objects:
https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/resource-controls

Reply:
Thanks @neolit123. I agree there is no need for a note or an issue. Just wanted to ensure I was not missing something.

Reply:
@jayunit100 per neolit123's comments here.

Windows has no concept of  `cgroup driver`. 
So I add a comment(not kubelet logs) here.

Reply:
Ok... the comment is good enough I guess :)